src/lib/layouts/Recipe.svelte

```diff
@@ -4,8 +4,11 @@
 	import { categories } from '$lib/stores/recipes';
 	import { page } from '$app/stores';
 	import Seo from '$lib/components/Seo.svelte';
+	import { format, formatDistanceToNow } from 'date-fns';
```

Comment:
perhaps we could use `Intl.DateTimeFormat` rather than adding an additional dependency?

Reply:
It's possible.
In fact, it's what I first wrote :sweat_smile: 

Let me change this